code_quality_tools/pmd.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
```

Comment:
Later there would be also one for Checkstyle & Findbugs so I thought a common folder would be adequate.